dist_tables/range_distribution.rst

```diff
@@ -34,61 +34,23 @@ Next, you can use the master_create_distributed_table() function to mark the tab
-    gzip -d github_events-2015-01-01-1.csv.gz
-
-Then, you merge both files into a single file by using the cat command.
-
-::
-
-    cat github_events-2015-01-01-0.csv github_events-2015-01-01-1.csv > github_events-2015-01-01-0-1.csv
-
-Next, you should sort the data on the repo_id column using the linux sort command.
-
-::
-
-    sort -n --field-separator=',' --key=4 github_events-2015-01-01-0-1.csv > github_events_sorted.csv
-
-Finally, you can split the input data such that no two files have any overlapping partition column ranges. This can be done by writing a custom script or manually ensuring that files don’t have overlapping ranges of the repo_id column.
-
-For this example, you can download and extract data that has been previously split on the basis of repo_id.
-
-::
-
-    wget http://examples.citusdata.com/github_archive/github_events_2015-01-01-0-1_range1.csv.gz 
-    wget http://examples.citusdata.com/github_archive/github_events_2015-01-01-0-1_range2.csv.gz 
-    wget http://examples.citusdata.com/github_archive/github_events_2015-01-01-0-1_range3.csv.gz 
-    wget http://examples.citusdata.com/github_archive/github_events_2015-01-01-0-1_range4.csv.gz
-    gzip -d github_events_2015-01-01-0-1_range1.csv.gz
-    gzip -d github_events_2015-01-01-0-1_range2.csv.gz
-    gzip -d github_events_2015-01-01-0-1_range3.csv.gz
-    gzip -d github_events_2015-01-01-0-1_range4.csv.gz
-
-Then, you can connect to the master using psql and load the files using the \\copy command.
-
-::
-
-    psql -h localhost -d postgres
-    SET citus.shard_replication_factor TO 1;
-    \copy github_events from 'github_events_2015-01-01-0-1_range1.csv' with csv
-    \copy github_events from 'github_events_2015-01-01-0-1_range2.csv' with csv
-    \copy github_events from 'github_events_2015-01-01-0-1_range3.csv' with csv
-    \copy github_events from 'github_events_2015-01-01-0-1_range4.csv' with csv
+The difference between range and append methods is that Citus’s distributed query planner has extra knowledge that the shards have distinct non-overlapping distribution key ranges. This allows the planner to push down more operations to the workers so that they can be executed in parallel. This reduces both the amount of data transferred across network and the amount of computation to be done for aggregation on the master.
 
-After this point, you can run queries on the range distributed table. To generate per-repository metrics, your queries would generally have filters on the repo_id column. Then, Citus can easily prune away unrelated shards and ensure that the query hits only one shard. Also, groupings and orderings on repo_id can be easily pushed down the workers leading to more efficient queries. We also note here that all the commands which can be run on tables using the append distribution method can be run on tables using range distribution. This includes \\copy, the append and shard creation UDFs and the delete UDF. 
```

Comment:
Because we are so thin on details, I figured we should just direct them to get in touch with us. We should really consider either improving our support, or just removing this section entirely at some point.